phable_cli/cli.py

```diff
@@ -186,9 +186,13 @@ def assign_task(ctx, task_ids: list[int], username: str | None):
     required=True,
     help="Name of destination column on the current project board",
 )
+@click.option(
+    "--milestone/--no-milestone",
+    default=True
```

Comment:
Let's add a `help` attribute, for documentation sake.
```suggestion
    default=True,
    help=(
        "If --milestone is passed, the task will be moved onto the project current associated "
        "milestone board, instead of the project board itself", 
    )
```

Reply:
Done